src/pmacfiltercontrol/zmqadapter.py

```diff
@@ -135,9 +153,12 @@ async def _process_message(self, message: Iterable[bytes]) -> None:
             print("No message")
 
     async def _process_response_queue(self) -> None:
+        """Process response message queue from the ZeroMQ stream."""
         print("Processing response queue...")
         running = True
         while running:
             resp = await self._read_response()
+            if resp is None:
+                continue
```

Comment:
This was down to a mypy complaint, that certain routes inside of `self._read_response()` didn't have a return statement. So I added a `return None` to the end to catch all other routes, which needed to be handled here

Reply:
But `self._read_response()` did return None before, it was just implicit. So it would have been calling `self._recv_message_queue.put_nowait(resp)` with None.

This definitely makes sense as a change, but I think it is a bug fix and sould be in its own commit.